sections/featured-collection.liquid

```diff
@@ -152,6 +153,12 @@
       "id": "show_vendor",
       "default": false,
       "label": "t:sections.featured-collection.settings.show_vendor.label"
+    },
+    {
+      "type": "checkbox",
+      "id": "show_rating",
+      "default": false,
+      "label": "t:sections.featured-collection.settings.show_rating.label"
```

Comment:
@melissaperreault Since this is per section, I am adding the translations to all the sections :)

Here is the commit so you can see: https://github.com/Shopify/dawn/pull/341/commits/b55d21fe7cccc45151835abe6d72334ffc437375